config/samples/example-test.sh

```diff
@@ -594,6 +594,10 @@ echo "\n"
 info "6.2.2 Add this private chartmuseum repository(basic auth enabled) to kubebb"
 kubectl apply -f config/samples/core_v1alpha1_repository_chartmuseum.yaml
 waitComponentStatus "kubebb-system" "repository-chartmuseum.nginx" "false"
+repoType=$(kubectl get repo repository-chartmuseum -nkubebb-system -ojson | jq -r '.metadata.labels."kubebb.repository.type"')
+if [repoType =="chartmuseum"]; then
```

Comment:
check "if not chartmuseum then exit 1" is a better choice.

Reply:
@zqq454224016  means you should exit with `1` which means fail if it is not `chartmuseum`